pkgs/development/haskell-modules/configuration-ghc-9.4.x.nix

```diff
@@ -194,6 +194,19 @@ in {
     hls-stylish-haskell-plugin = null;
   };
 
+  http-api-data = super.http-api-data_0_5;
+  attoparsec-iso8601 = super.attoparsec-iso8601_1_1_0_0;
+
+  swagger2 = dontCheck super.swagger2;
```

Comment:
What's the story here? Can you add a comment/link an upstream issue if applicable?